.github/workflows/workflow.yml

```diff
@@ -0,0 +1,153 @@
+name: Country Template
+
+on: [ push ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.12
+      - name: Cache build
+        id: restore-build
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ github.ref }}${{ env.pythonLocation }}-build-${{ hashFiles('setup.py') }} # Cache the entire build Python environment
+      - name: Build package
+        if: steps.restore-build.outputs.cache-hit != 'true'
+        run: make build
+      - name: Cache build release
+        id: restore-build-release
+        uses: actions/cache@v2
+        with:
+          path: dist
+          key: ${{ github.ref }}${{ env.pythonLocation }}-build-release-${{ hashFiles('setup.py') }}
+
+  lint-files:
+    runs-on: ubuntu-latest
+    needs: [ build ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all the tags
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.12
```

Comment:
Do we really need to specify down to the patch version? I would prefer if we could simply define `3.7` and work on the latest patch version 🙂

Reply:
Yes. Because the name of the environment in the cache depends on the patch as well. And in `Openfisca-France' there were issues where the build used 3.7.12 and the test used 3.7.11 and it couldn't find the right cache 🤔